src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -0,0 +1,75 @@
+import { html } from 'lit-element';
+import sortAscendingIcon from '../assets/icon_sort_ascending.js';
+import sortDescendingIcon from '../assets/icon_sort_descending.js';
+
+import './volumes.js';
+
+export default class VolumesProvider {
+
+  constructor(optionChange, baseHost, bookreader) {
+    this.optionChange = optionChange
+    this.component = document.createElement('viewable-files');
+
+    const files = bookreader.options.multipleBooksList?.by_subprefix
+    this.viewableFiles = Object.keys(files).map(item => files[item]);
+    this.volumeCount = Object.keys(files).length;
+    this.isFirstLoad = true
```

Comment:
SUGGESTION: Add semicolon to end of lining. BTW, are you using an auto code-formatter for JS in your code editor? It would catch omissions like this.